styled/store.js

```diff
@@ -107,23 +107,42 @@ export const styles = StyleSheet.create({
   input: {
     fontFamily: 'poppins-regular',
   },
-  chip: {
+  tagChip: {
     backgroundColor: Colors.lightestGreen,
     color: Colors.darkerGreen,
     marginRight: 6,
-    marginBottom: 4,
```

Comment:
Wait we need this! That's A & S looked super weird with 2 lines of tags

Reply:
wait I only removed it because it seems very odd to have both `marginVertical` and `marginBottom` and I expected that `marginVertical` overrode the latter - why wasn't it just `marginBottom` and `marginTop`?

Reply:
I had to set `marginVertical` to 0 to override the defaults https://github.com/callstack/react-native-paper/blob/master/src/components/Chip.tsx#L333-L338 because it wasn't shrinking to my custom height (see my message in #ask), and I needed it to have 0 top margin.
![image](https://user-images.githubusercontent.com/21160510/79701021-99589200-824e-11ea-89d0-f31f8631aac9.png)


There's probably a cleaner way to do this but I'm not super sure what combination of these margins work best.

Reply:
ok re-ordered them for my sanity but if it works it works!